app/controllers/restaurants_controller.rb

```diff
@@ -28,12 +28,12 @@ def create
     @restaurant = Restaurant.new(restaurant_params)
 
     respond_to do |format|
-      if @restaurant.save
-        format.html { redirect_to @restaurant, notice: 'Restaurant was successfully created.' }
-        format.json { render :show, status: :created, location: @restaurant }
-      else
-        format.html { render :new }
-        format.json { render json: @restaurant.errors, status: :unprocessable_entity }
+  if @restaurant.save
```

Comment:
@ross please fix identation
```suggestion
       if @restaurant.save
```

Reply:
:wave: @claracodes, I'm probably not the Ross you're looking for 😁

Reply:
Oh so sorry! I was just showing my students GitHub - and one of them called Ross 😆

Reply:
No worries. 😄